docs/fairness-indicators.md

```diff
@@ -0,0 +1,98 @@
+across any size of use case, in the TensorBoard environment or in
+[Colab](https://github.com/tensorflow/fairness-indicators).
+
+## Requirements
+
+To install Fairness Indicators for TensorBoard, run:
+
+```
+python3 -m virtualenv ~/tensorboard_demo
+source ~/tensorboard_demo/bin/activate
+pip install --upgrade pip
+pip install tensorboard_plugin_fairness_indicators
+pip install "tensorflow_model_analysis>=0.15.1"
+pip uninstall -y tensorboard tb-nightly
+pip install --upgrade tb-nightly
+```
+
+## Demo
+
+If you want to test out Fairness Indicators in TensorBoard, you can download
+sample TensorFlow Model Analysis evaluation results (eval_config.json, metrics
+and plots files) and a `demo.py` utility from Google Cloud Platform,
+[here](https://console.cloud.google.com/storage/browser/tensorboard_plugin_fairness_indicators/).
+(Checkout [this](https://cloud.google.com/storage/docs/downloading-objects)
+documentation to download files from Google Cloud Platform). This evaluation
+data is based on the
+[Civil Comments dataset](https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification),
+calculated using Tensorflow Model Analysis's
+[model_eval_lib](https://github.com/tensorflow/model-analysis/blob/master/tensorflow_model_analysis/api/model_eval_lib.py)
+library. It also contains a sample TensorBoard summary data file for reference.
+See the
+[TensorBoard tutorial](https://github.com/tensorflow/tensorboard/blob/master/README.md)
+for more information on summary data files.
+
+The `demo.py` utility writes a TensorBoard summary data file, which will be read
+by TensorBoard to render the Fairness Indicators dashboard. Flags to be used
+with the `demo.py` utility:
+
+-   `--logdir`: Directory where TensorBoard will write the summary
+-   `--eval_result_output_dir`: Directory containing evaluation results
+    evaluated by TFMA (downloaded in last step)
+
+Run the `demo.py` utility to write the summary results in the log directory:
+
+`python demo.py --logdir=<logdir>/demo
+--eval_result_output_dir=<eval_result_dir>`
+
+Run TensorBoard:
+
+Note: For this demo, please run TensorBoard from the same directory where you
+have downloaded the evaluation results.
+
+`tensorboard --logdir=<logdir>`
+
+This will start a local instance. After the local instance is started, a link
+will be displayed to the terminal. Open the link in your browser to view the
+Fairness Indicators dashboard.
+
+## Usage
+
+To use the Fairness Indicators with your own data and evaluations:
+
+1.  Train a new model and evaluate using
+    `tensorflow_model_analysis.run_model_analysis` or
+    `tensorflow_model_analysis.ExtractEvaluateAndWriteResult` API in
+    [model_eval_lib](https://github.com/tensorflow/model-analysis/blob/master/tensorflow_model_analysis/api/model_eval_lib.py).
+    For code snippets on how to do this, see the Fairness Indicators colab
+    [here](https://github.com/tensorflow/fairness-indicators).
+
+2.  Run the `demo.py` file (download
```

Comment:
This bullet point feels weird to see under the `Usage` header. We already discuss the demo case under `Demo`. Should you perhaps instead talk about the summary API?

Reply:
Done